app/services/file_identifier_service.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+# Identifies a file using Siegfried.
+class FileIdentifierService
+  class FileIdentifierServiceError < StandardError
```

Comment:
Trivial naming thing: since this exception class is defined in `FileIdentifierService`, could it be `Error`? That way the namespaced reference would be `FileIdentifierService::Error` (without the repetition of the containing class). Like I said, trivial.

Reply:
Thanks. Fixed.